### PR TITLE
Fail build on ERROR commit status & add logging to Jenkins output to mirror Slack notifications.

### PR DESCRIPTION
**Pull request by ElijahLynn**
## Description

Was hard to debug why a job was acting as a "no-op". Logging was only
getting sent to Slack channel. In our case the commit status was "error"
and the job had logic to not go any further, but we didn't know that
until we looked at the Slack channel for the log message.

https://va-gov.atlassian.net/browse/VAGOV-6361

## Testing done
Tested a replay here 
http://jenkins.vfs.va.gov/job/builds/job/vets-website-content-vagovstaging/129/console

## Screenshots
image

image

## Acceptance criteria
- [ ]

## Definition of done
- [x] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ElijahLynn** [comment]
Oh, the "abort" step should also mark job as a failure. Working on that now. 

image

**Comment by ElijahLynn** [comment]
Pushed - 
image

File: Jenkinsfile.content
Comment (nathanhruby) [inline Jenkinsfile.content]: Did you mean to delete this section?

Comment (ElijahLynn) [reply Jenkinsfile.content]: No, I didn't mean to delete that. I deleted it for testing and copy/pasted into Jenkins. Thanks!

Comment (ElijahLynn) [reply Jenkinsfile.content]: Fixed, rebased, force pushed.

File: Jenkinsfile.content
Comment (ElijahLynn) [inline Jenkinsfile.content]: I just tested this format of interpolating this variable and it works and is more readable.

**Comment by ElijahLynn** [comment]
This is 100% Jenkins code, so it may not make much sense for the FE team to review and that is why I tagged Wyatt and Nathan for review.

**Review by nathanhruby — APPROVED** [review]
Seems good to me, but I am not a codeowner here
You Have No Power Here

**Review by cvalarida — APPROVED** [review]

**Comment by ElijahLynn** [comment]
Nice, confirmed this is working after merge!
image

**Pull request merged**
